### ISSUE TITLE
Garden Crossbreeding Suggestion

### ISSUE THREAD

**Issue by CardinalBirb** [prinzstani/CookieBot#66]
Is it possible for the bot to stop harvesting Elderworts (and any other immortal plant) whenever they're "about to die" and trying to crossbreed? I've been watching the bot try and get crossbreeds with Elderwort for a while now and it always gets harvested. It's a very expensive plant and also it would help to crossbreed immensely as the lifespans of the plants don't really line up.

Also, a bit less important is it possible to get the bot to plant the crops in a way so that they reach maturity (crossbreeding age) synchronized?

**Comment by prinzstani** [comment]
I agree. I have also noticed that strange behaviour. This should definitely be fixed, even though it seems to work over a longer time.
Synchronizing is in place for the juicy queenbeet, but not for the other plants. For most plants, it does not make too much sense as the growing times are vastly different, but maybe for the everdaisy it should also be synchronized.

**Comment by CardinalBirb** [comment]
I see, thank you!
